protocol/avdl/keybase1/common.avdl

```diff
@@ -193,6 +193,18 @@ protocol Common {
     MOBILE_1
   }
 
+// these enum values need to match the ones in
+// keybase/lib_public/public_constants.ts ->
+// public_constants.device.type
+  enum DeviceTypeV2 {
```

Comment:
talked it over with @mmaxim, he's still nervous this is going to break something and be hard to debug. There's no way in a review I can be certain it won't break things.


let's stick with the original original plan:

```
@typedef ("string") record DeviceTypeV2 {}
```

Reply:
In general changing the AVDL in a way that's no backwards compatible is a huge can of worms and we never do it.